### ISSUE TITLE
Compiling mosh for RHEL8

### ISSUE THREAD

**Issue by moonlight16** [mobile-shell/mosh#1149]
I'm trying to get mosh to compile on RHEL8.

I'm getting the following error when running `configure`:

```
[root@jc-rhel81 mosh]# ./configure
....
configure: error: Package requirements (protobuf) were not met:

Package 'protobuf', required by 'virtual:world', not found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables protobuf_CFLAGS
and protobuf_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```

I installed protobuf using the standard `yum install protobuf protobuf-compiler`:

```

[root@jc-rhel81 mosh]# sudo yum list | grep protobuf
protobuf.x86_64                                        3.5.0-13.el8                                        @Local-Appstream
protobuf-c.x86_64                                      1.3.0-6.el8                                         @Local-Appstream
protobuf-compiler.x86_64                               3.5.0-13.el8                                        @Local-Appstream
protobuf.i686                                          3.5.0-13.el8                                        Local-Appstream
protobuf-c.i686                                        1.3.0-6.el8                                         Local-Appstream
protobuf-c-compiler.i686                               1.3.0-6.el8                                         Local-Appstream
protobuf-c-compiler.x86_64                             1.3.0-6.el8                                         Local-Appstream
protobuf-c-devel.i686                                  1.3.0-6.el8                                         Local-Appstream
protobuf-c-devel.x86_64                                1.3.0-6.el8                                         Local-Appstream
protobuf-compiler.i686                                 3.5.0-13.el8                                        Local-Appstream
protobuf-lite.i686                                     3.5.0-13.el8                                        Local-Appstream
protobuf-lite.x86_64                                   3.5.0-13.el8                                        Local-Appstream
python3-protobuf.noarch                                3.5.0-13.el8                                        Local-Appstream
```

I shouldn't have to use it, since its installed in the standard path.  

I'm not exactly sure what to put in the PKG_CONFIG_PATH.  

Wondering if anyone can help point me in the right direction.

**Comment by eminence** [comment]
I don't quite understand this error.  Normally if protobuf is missing, you'd see an error like:

    configure: error: cannot find protoc, the Protocol Buffers compiler

And installing both `protobuf` and `protobuf-compiler` should be sufficient.

Can you attach the `config.log` file that would have been created after running `./configure` ?

**Comment by andersk** [comment]
You are missing the `protobuf-devel` package, which is in the optional CodeReady Linux Builder repository in RHEL 8.

**Comment by moonlight16** [comment]
> You are missing the `protobuf-devel` package, which is in the optional CodeReady Linux Builder repository in RHEL 8.

I tried installing the following packages, but I still get the same error.

`sudo yum install  protobuf-c-devel protobuf-c protobuf-c-compiler`

**Comment by moonlight16** [comment]
> You are missing the `protobuf-devel` package, which is in the optional CodeReady Linux Builder repository in RHEL 8.

First, I found my no-cost subscription was expired, so I had to first renew it on the access.redhat.com.  Then I could re-register my system and find subscriptions.

Second, I had to enable the following subscription:
`subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms`

Last, I could install `protobuf-devel`:
`sudo yum install protobuf-devel`

Note: the default repos show `protobuf-c-devel`, this is not the correct one.

Once I did that, I could successfully get through `./configure`.
